Library/Formula/prometheus.rb

```diff
@@ -0,0 +1,28 @@
+class Prometheus < Formula
+  desc "Service monitoring system and time series database"
+  homepage "http://prometheus.io"
+  url "https://github.com/prometheus/prometheus.git",
+    :tag => "0.17.0",
+    :revision => "e11fab35d76d19c5c49b7d85e28275f894d3ada4"
+  sha256 "363400cd2e1167a1a62d99fc344b477c97c289876d70d39318d82855b6eaf2c2"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
+    mkdir_p buildpath/"src/github.com/prometheus"
+    ln_sf buildpath, buildpath/"src/github.com/prometheus/prometheus"
+
+    system "make", "build"
+    bin.install "promtool"
+    bin.install "prometheus"
+    libexec.install "consoles"
+    libexec.install "console_libraries"
+  end
+
+  test do
+    system "#{bin}/prometheus", "--version"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!